bookish/Services/auxiliary_functions.py

```diff
@@ -0,0 +1,31 @@
+import hashlib
+from bookish.models.users import Users
+
+
+def hashing_algorithm(string):
+    result = hashlib.md5(string.encode('utf-8')).hexdigest()
+    return result
+
+
+def check_auth_token(token):
+    """
+    :param token:
+    :return: true -> token valid
+             false -> token expired/invalid
+    """
+
+    all_users = Users.query.all()
+    for user in all_users:
+        print(user)
+        if user.user_token == token and user.user_token_expire > datetime.datetime.now():
+            return True
+    return False
+
+
+def check_if_authenticated(auth_token):
+    try:
+        authenticated_status = check_auth_token(auth_token)
+        if not authenticated_status:
+            raise {"status": "Error", "error_message": "Not authenticated", "UI_message": "Not authenticated"}

```

Comment:
I like that you return a nicer message to display to the user😄 👍 

Very tiny nitpick, we tipically refer to them as `user_friendly`.

Reply:
We could also return a http status code for when you aren't logged / you don't have permissions.